src/sophys/common/devices/motor.py

```diff
@@ -144,7 +144,7 @@ def MotorGroup(prefix, motors_suffixes, name, **kwargs):
             suffix = suffix[0]
             deviceClass, comp_kwargs = _create_virtual_controllable_motor(suffix[1])
 
-        components[key] = Component(deviceClass, prefix=suffix, **comp_kwargs)
+        components[key] = Component(deviceClass, suffix=suffix, **comp_kwargs)
```

Comment:
where is this `suffix` parameter defined? I changed it in the previous PR because I couldn't find it being used anywhere.

Reply:
`suffix` is the default `Component` argument, usually passed as a positional arg.